src/WebJobs.Script/Environment/EnvironmentExtensions.cs

```diff
@@ -156,6 +156,17 @@ public static bool IsWindowsConsumption(this IEnvironment environment)
             return string.Equals(value, ScriptConstants.DynamicSku, StringComparison.OrdinalIgnoreCase);
         }
 
+        /// <summary>
+        /// Gets a value indicating whether the application is running in a Windows or Linux Consumption (dynamic)
+        /// App Service environment.
+        /// </summary>
+        /// <param name="environment">The environment to verify</param>
+        /// <returns><see cref="true"/> if running in a Windows or Linux Consumption App Service app; otherwise, false.</returns>
+        public static bool IsConsumption(this IEnvironment environment)
```

Comment:
Rename to IsConsumptionSku? ( that is how this method is named in dev branch)

Reply:
done!